.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
This file should probably not be added

Reply:
Why not? `root=true` means that this .editorconfig is the root of your configuration for this repository. If you do not specify it and have an editorconfig at a higher level on your local file system, that means that that other editorconfig's configuration is also taken into account, we do not want that. It is not related to the root user or anything like that. See https://editorconfig.org

Reply:
And if it is not about the root part, the editorconfig is a fairly standard way of noting the coding style in a project, most modern editors (VSC, BBEdit, PyCharm, Atom, etc) support this. It ensures that when you indent xml using the tab key that it actually results in a tab and not spaces.

Reply:
Sorry, it was about the file in general, I just did not see a way to make a file level comment.

The comment was simply because I have intentionally avoided adding any editor/preferences files, so I assumed that this was unintentionally added as a personal settings file that got caught up in other changes

Reply:
Personal preferences for IDEs etc should go in your home directory (whatever OS you're running on), but this one is about code format - if there was some way of adding xml linting to it I'd be all for that too (I know IDEs can handle that, but having "follow these rules globally" is always better than having everyone using different rules).

Even though I always prefer tabs over spaces, you cannot rely on git being able to convert them properly, so without this file you end up with mismatched indentations and "dirty" PRs :-/

Reply:
In general, the editor settings are not in the repo, you're correct. However, it's possible to add settings specific to a repository, which I have for this one and, at least for me, are stored in the repository. That's what I was referring to having intentionally not added.

I do understand that this is for coding style. I was just periodically running everything through reformatting while I was updating the repo anyway, so this was never really a problem or concern for me, especially since I was working alone on this version to get it up to date (really just for my own use) and that the format of the data was, ultimately, a secondary goal to having it up to date and useable.

As for xml linting, I do have that as part of the CI checks to ensure that broken xml is not added, it is not 100%, but nothing is and it has done pretty well for me. These checks also make sure that the compendium building is not broken by the additions.

Honestly, part of my problem with this was just that it was added as part of a fairly large set of changes and was not really part of them, in addition to simply believing that it was an unnecessary file that had been accidentally committed.